install

```diff
@@ -3,16 +3,11 @@
 : ${GIT_PROJECT:=cloudbreak-deployer}
 
 main() {
-    local redirect=$(
-    curl -s \
-        -o /dev/null \
-        -w "%{redirect_url}" \
-        https://github.com/sequenceiq/cloudbreak-deployer/releases/latest
-    )
-    local latest=${redirect##*v}
+    mkdir -p cloudbreak-deployer
+    cd cloudbreak-deployer
     local osarch=$(uname -sm|tr " " _)
 
-    local url="https://github.com/${GIT_ORG}/${GIT_PROJECT}/releases/download/v${latest}/${GIT_PROJECT}_${latest}_${osarch}.tgz"
+    local url="https://github.com/${GIT_ORG}/${GIT_PROJECT}/releases/download/v1.1.0/${GIT_PROJECT}_1.1.0_${osarch}.tgz"
```

Comment:
Please introduce a `CBD_VERSION` variable to the top of the script, so we can automate the replacement